drivers/staging/vc04_services/bcm2835-camera/mmal-vchiq.c

```diff
@@ -1934,6 +1945,9 @@ int vchiq_mmal_init(struct vchiq_mmal_instance **out_instance)
 
 	params.callback_param = instance;
 
+	instance->bulk_wq = alloc_ordered_workqueue("mmal-vchiq",
+						    WQ_MEM_RECLAIM);
```

Comment:
Should an allocation failure be acted upon?

Reply:
Yes. Adding it.